### ISSUE TITLE
Hotkey for opening popup

### ISSUE THREAD

**Issue by mike-tn** [2alin/tabFx#9]
Please add a keyboard shortcut to open popup window.
User can assign it in Add-ons manager->Manage extension shortcut.

**Comment by 2alin** [comment]
Yup, accessibility is something that I need to consider in the extension. 
I have some serious code cleaning and refactor on mind, as preparation for new features.
